examples/interactive_bluefog_helloworld.ipynb

```diff
@@ -0,0 +1,634 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "engine": 2,
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%px  --targets 1:3\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "plt.semilogy(mse)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What happens after you run `ibfrun`\n",
+    "\n",
+    "If you are curious about what happened after running the ibfrun, the following figure should give you a good overview.\n",
+    "\n",
+    "<img src=\"./images/ibfrun.png\" alt=\"ibfrun\" width=\"780\"/>\n",
+    "\n",
+    "When you start the jupyter notebook, you will create a IPython client, which will keep recieving and executing the command from the notebook. `ibfrun` will create a controller and bunch of workers (it is just IPython Kernel actually) under the MPI environment. The controller knows how to communicate with the IPython client and the workers. When you execute the code under `px` environment, what happens:\n",
```

Comment:
- bunch -> a bunch
- (it is just IPython Kernel actually) -> (They are actually IPython Kernels)
- what happens -> the following procedure happens